src/main/app/claims/models/claim.ts

```diff
@@ -87,12 +89,71 @@ export class Claim {
     return this.responseDeadline.diff(MomentFactory.currentDate(), 'days')
   }
 
+  get isFullAdmission (): boolean {
+    return this.response && this.response.responseType === ResponseType.FULL_ADMISSION
+  }
+
+  get isPartialAdmission (): boolean {
+    return this.response && this.response.responseType === ResponseType.PART_ADMISSION
+  }
+
+  get isPayImmediately (): boolean {
+    const response = (this.response as FullAdmissionResponse | PartialAdmissionResponse)
```

Comment:
Just as a point of discussion, not for this PR:
Would it be helpful to generalise `FullAdmissionResponse` and `PartialAdmissionResponse` into an abstract `AdmissionsResponse`? This `(response as FullAdmissionResponse | PartialAdmissionResponse)` is found all over the place...